src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -198,10 +204,12 @@ export class TemplateActionService extends SyncServiceBase {
         // if there is no http or https then there is no : to separate and we only have a key in the url. This
         // case then adds '//' to the key so it recognises it as external and not local
         // This removes the need to have http or https in the url.
-        if (!value) {
-          value = "//" + key;
-        }
-        return this.templateNavService.handleNavActionExternal(value);
+        // make a prior check for if the key starts with https:// and if it does then it is a url and should be opened
```

Comment:
todo (blocking): This comment is out-dated, as this check isn't being performed. I would suggest replacing all the comments from lines 201 to 207 with a new comment that summarises what's happening on lines 208-212 and why (i.e. because URL strings get separated by the parser at the ":" character in some cases)